modules/01_macro/singleSectorGr/equations.gms

```diff
@@ -41,6 +41,8 @@ qm_budget(ttot,regi)$( ttot.val ge cm_startyear ) ..
         )$( ttot.val ge max(2010, cm_startyear) )
       )
     )
+  + sum(tradeSe, pm_MPortsPrice(ttot,regi,tradeSe) * vm_Mport(ttot,regi,tradeSe)) 
```

Comment:
The corresponding lines for PE trade have the trade (transport) costs here, not the price of the good. Is this necessary because we do not implement SE trade via the nash markets yet?

Reply:
I will double check that. Thks for the comment!

Reply:
Renato, the original trade cost implementation was meant as a kind of iceberg cost, i.e. costs that have to be paid by someone, but there is no beneficiary. All other revenues from the PE trade is accounted for via the intertemporal budget constraint. If you include SE trade which have a wedge in export and import prices, you are probably doing right to include this kind of producer revenue in the budget equation. But I am not fully sure about the implied consequences for the overall accounting. Is the convergence fine?

Reply:
Thks @mleimbach
So far the convergence is fine because we are only adding very small capacities for trade SE. As mentioned in another comment also in this pull request, this needs to be updated for a more wider secondary energy trading scenario, but this will most probably be the challenge for a newcomer PhD/PostDoc.